### PR TITLE
actually upgrade to 1.4

### PR DESCRIPTION
**Pull request by jamesgorrie**
For some reason, jspm didn't update my config?
I will try to find out why, but a reset worked.

### CODE REVIEW

File: kahuna/public/js/components/gu-date-range/gu-date-range.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ guDateRange.directive('guDateRange', [function () {
             });
 
             $scope.$watch('pikaStartValue', function (pikaStartValue) {
-                var date = pikaStartValue && new Date(pikaStartValue);
+                var date = (pikaStartValue && new Date(pikaStartValue)) || new Date();
```
Comment (jamesgorrie) [inline kahuna/public/js/components/gu-date-range/gu-date-range.js]: This is currently passing an empty string about, which is dealt with silently until now.

Comment (theefer) [inline kahuna/public/js/components/gu-date-range/gu-date-range.js]: Is this causing an error? Have you tested that the behaviour is still as expected?

cc @akash1810 who worked on that bit.

**Comment by theefer** [comment]
:+1:

File: kahuna/public/js/components/gu-date-range/gu-date-range.js (hunk 1 above)
Comment (jamesgorrie) [inline kahuna/public/js/components/gu-date-range/gu-date-range.js]: Yip - everything is also working as expected.
Passing an empty string to `Date` should error irrespective, so this code is more typesafe anyway.

File: kahuna/public/config.js [hunk 2]
```diff
@@ -11,25 +11,25 @@ System.config({
...
     "jcrop": "github:tapmodo/Jcrop@0.9.12",
     "jquery": "github:components/jquery@2.1.1",
-    "mixpanel-js": "github:mixpanel/mixpanel-js@2.4.2",
-    "moment": "github:moment/moment@2.9.0",
+    "mixpanel-js": "github:mixpanel/mixpanel-js@2.5.2",
+    "moment": "github:moment/moment@2.10.5",
     "pandular": "npm:pandular@0.1.5",
-    "pikaday": "github:dbushell/Pikaday@1.3.2",
-    "raven-js": "github:getsentry/raven-js@1.1.18",
+    "pikaday": "github:dbushell/Pikaday@1.3.3",
```
Comment (jamesgorrie) [inline kahuna/public/config.js]: This was updates which doesn't accept bad dates any moer.

**Pull request merged**